dts/arm/st/f1/stm32f107.dtsi

```diff
@@ -16,5 +16,17 @@
 			interrupts = <56 0 57 0 58 0 59 0 60 0>;
 			status = "disabled";
 		};
+
+		mac: ethernet@40028000 {
+			compatible = "st,stm32-ethernet";
+			reg = <0x40028000 0x8000>;
```

Comment:
could you check as the RefMan0008  gives 0x40028000 - 0x40029fff (0x2000)

Reply:
Right. I copied over from `stm32f207.dts`, assuming it was the same. In fact, for STM32F207, it is defined from 0x4002 8000 to 0x4002 93FF, and for STM32F107 it is 0x4002 8000 to 0x4002 9FFF. Both Ethernet register maps give last register at offset 0x1054.